### ISSUE TITLE
Add missing Aria Attribute for Disclosure

### ISSUE THREAD

**Issue by ghost** [jwstegemann/fritz2#753]
Currently the `aria-controls` property is missing on the `disclosureButton` - it is only rendered inside the `disclosureCloseButton`, which is not enough.
